css/style.css

```diff
@@ -5,6 +5,68 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+//navbar is a list of links, but i do not plan to make them working
+ul{ text-align:right; }
+li { 
+    padding:10px;
+    display: inline-block; 
+    justify-content: center;
+    
+    }
+    li:hover {
+        font-style:bold;
+        cursor: pointer;
+        color: #910909;
+    }
+navbar{
+    display:flex;
+    flex-basis:250vw;
+}
+.container{
+    display:flex;
+    flex-direction: row;
+    justify-content:space-around;
+    width:80%;
+}
+h1{
+    font-weight: lighter;
+    font-size: 2.75rem;
+}
+.aside{
+    position: relative;
+    text-align: center;
+    color: white;
+  }
+  .centered {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  button {
+    width: 9.688rem;
+    background-color: #f15a29;
+    border-color: #f15a29;
+    text-shadow: none;
+    border-style: none;
+    border-radius: 4px;
+    padding: 1rem 1.5rem;
+    color: white;
+    font-size: 1.125rem;
+}
+h2{ text-align:center;
+    font-size: 1.75rem;
+    margin: 1.625rem 0 3.125rem;
+    font-weight: 300;
+}
+.icontainer{
```

Comment:
Try and add a a little more spacing above and below this section to give it some breathing space.